Document sync extensions (#24)
mirumee/ariadne-website: docs/extensions.md

Ariadne implements simple extension system that allows developers to inject custom python logic into the query execution process.

> > At the moment, Subscriptions are not supported. Please see notes on WSGI implementation.

Reviewer: ```suggestion
> At the moment adding extensions to subscriptions is not supported.
```

                "execution": self.start_timestamp - self.end_timestamp
            }
```

> ## WSGI extension implementation

Reviewer: Please perepend headers with two blank lines:

```suggestion

## WSGI extension implementation
```
